backend/sftp/options.go

```diff
@@ -27,7 +27,7 @@ type Options struct {
 	KnownHostsFile     string              `json:"knownHostsFile,omitempty"` // env var VFS_SFTP_KNOWN_HOSTS_FILE
 	KnownHostsString   string              `json:"knownHostsString,omitempty"`
 	KeyExchanges       []string            `json:"keyExchanges,omitempty"`
-	Ciphers            []string            `json:"cihers,omitempty"`
+	Ciphers            []string            `json:"ciphers,omitempty"`
```

Comment:
oh wow... that's a good catch... this is probably breaking for someone, somewhere.

Reply:
Can we support reading both fields but always write the correct spelling when marshaling moving forward?